librz/arch/isa/msp430/msp430_il_jmp_utils.inc

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2024 Mostafa Mahmoud <ubermenchun@gmail.com>
```

Comment:
Is there a specific reason this is an `inc` file? Can't it be its own header file with `static inline` functions?
`inc` files tend to break the syntax highlighting and is really just useful for huge amounts of generated code.

Reply:
Hmmm, I usually think of  ` .h`s as files that contain "non-logic" code: declarations, type definitions, constants, and so on. I think of `.c`s as files that contain logic and should be seperately compiled, but `inc`s are those that contain logic but shouldn't be seperately compilable, as they're considered just fragment of a single "owning" `.c` file.

That's my "head cannon", so to speak. But I can change it anyway if it conflicts with any convention or aesthetic preference.

Reply:
I would vote to move it to a normal .c .h form. In my experience working with .inc files is always a little annoying in one or another aspect. @XVilka @DMaroo